core/cas-server-core/src/main/java/org/apereo/cas/CentralAuthenticationServiceContext.java

```diff
@@ -33,7 +34,8 @@
 
     private final PrincipalFactory principalFactory;
 
-    private final TicketRegistry ticketRegistry;
+    @Setter
+    private TicketRegistry ticketRegistry;
```

Comment:
This is not a good idea. Fields must be left as immutable.

Reply:
Will remove